src/app/pages/ArticlePage/ArticlePage.styles.ts

```diff
@@ -98,7 +98,7 @@ export default {
         marginBottom: `${spacings.FULL}rem`,
         padding: `${spacings.DOUBLE}rem`,
       },
-      '[amp-x-topStoriesExperiment="show_at_halfway"] &': {
+      '[amp-x-topStoriesExperiment*="show"] &': {
```

Comment:
Commenting since this seems a little out of place if just inspecting the diff.

This conveniently catches all the variants as they all start with `show_at_`. It is applied to the original Top Stories section in the secondary column to hide it whenever a variant that shows it in the article body is shown.

Reply:
Do you still need this if you've added the css for each variant in `src/app/pages/ArticlePage/experimentTopStories/index.styles.ts`?

Reply:
Gotcha!